Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -32,17 +32,17 @@ public class NetworkBehaviour : MonoBehaviour
         protected readonly List<SyncObject> m_SyncObjects = new List<SyncObject>();
 
         // NetworkIdentity component caching for easier access
-        NetworkIdentity m_netIdentity;
+        NetworkIdentity networkIdentity;
         public NetworkIdentity netIdentity
```

Comment:
that's a little weird since we have networkIdentity and netIdentity then.
how about _networkIdentity?